website/src/views/home.js

```diff
@@ -135,6 +154,14 @@ function Home() {
                             </Grid>
                         </Box>     
                     }
+                    { localStorage.getItem('token') &&
+                        <>
```

Comment:
This isn't something to fix, per-se, but generally its good to avoid fragments when possible.  I would tend to just opt for a `Box` or a `div` because its more extensible and easier to debug.

Reply:
replaced fragment element with a Box element